tests/testthat/test-modify-model-field.R

```diff
@@ -354,3 +354,50 @@ test_that("remove_tags() works correctly [BBR-MMF-022]", {
   )
   expect_identical(new_mod[[YAML_TAGS]], ref_tags)
 })
+
+test_that("Checking that the star_add function is adding boolean data to YAML[BBR-MMF-023]", {
```

Comment:
* missing space before test tag
  * `star_add` -> `add_star`
  * You are using the same test tag/id for multiple tests.  This is fine for related tests like these, and means they are rolled up into a single description for the validation docs.  However, be careful because all the non-shared parts of the description are lopped off.  In this case it'd be "Checking that the star_" or Checking that the" (I need to look at the code to see if it's a word-based split).  In either case, that's pretty uninformative for a test description and will look incomplete to anyone reading the validation docs.  For this particular case, I'd say just use different test tags (e.g., "add_star() stars a model [BBR-MMF-023]" and "remove_star() unstars a model [BBR-MMF-024]").